### ISSUE TITLE
Default parameter and variables values are broken

### ISSUE THREAD

**Issue by bryce-carson** [ashokkrish/episim#61]
Despite the code below, default parameter values and variables are not updating properly.

https://github.com/ashokkrish/episim/blob/e94fe2905aafc46e68f7ab011b9d061a139d3f94/server.R#L234-L238

The NOTE in the quotation below is out of date. There is no case_when now.

https://github.com/ashokkrish/episim/blob/e94fe2905aafc46e68f7ab011b9d061a139d3f94/server.R#L29-L41

**Comment by bryce-carson** [comment]
The condition has the wrong effect for the intention. The first step in fixing this should be to remove it.

https://github.com/ashokkrish/episim/blob/91fe4e955163d2b5a6dba9f03dc869c68c1874fa/server.R#L32

**Comment by bryce-carson** [comment]
@kle6951, I apologize, but I had an itching to work on the code and I mostly fixed this issue. I recommend, to get your practice in with R, that you use the prior commit and try to see if you come up with a similar solution.

There is still work to do to fully resolve this issue; see the commit message in the commit that referenced this issue just now.

**Comment by bryce-carson** [comment]
These are the issues I identified in the code while working on this issue.

- [ ] `updateNumericInputs` now uses iwalk rather than mapply, and will iterate over the provided values names and values as intended. The previous implementation did not work at all; I wonder why I wrote it that way, because I do recall doing a test in the console. :shrug:
- [ ] `updateNumericInputs`, while a reactive, does not automatically wrap all functions called from inside it in a reactive context. Frames can exist outside its reactive context, despite their parent or calling frame originating from within that reactive context. The solution is a `with` and some `reactiveValues` calls.
- [ ] The defaultInputValues dataframe is, for some strange reason, not being filtered down to a single row despite enough information being provided to get a unique row from, say, SIRS.
